docs/primitives/a-sky.md

```diff
@@ -4,6 +4,18 @@ type: primitives
 layout: docs
 parent_section: primitives
 source_code: src/extras/primitives/primitives/a-sky.js
+examples:
```

Comment:
Thanks! I might remix them to rename like `aframe-*` and then format the code a bit to match the other ones (indents, less whitespace).

Reply:
Yes you are in fact right, it's just I haven't gotten used to that structure just yet. I'm sorry for giving you extra work